pymc3/tests/test_distributions.py

```diff
@@ -958,6 +966,14 @@ def test_laplace(self):
             lambda value, mu, b: sp.laplace.logcdf(value, mu, b),
         )
 
+    def test_laplace_asymmetric(self):
+        self.pymc3_matches_scipy(
+            AsymmetricLaplace,
+            R,
+            {"b": Domain([0, 1, inf]), "kappa": Rplus},
```

Comment:
Since both `b` and `kappa` parameters have same domains, why not use `Rplus` Domain for `b` as well?

```suggestion
            {"b": Rplus, "kappa": Rplus},
```

Reply:
It depends if  I modify the previous suggestion. Since I was  not using `b`  in for the unit test, I thought of assigning `b` a constant value 1.
If I should  make all these changes, do let me know

Reply:
Yes. We need to have test logpdf including `b` as well since it can be parametrized in the model.